aten/src/ATen/native/LinearAlgebra.h

```diff
@@ -18,12 +18,4 @@ DECLARE_DISPATCH(addr_fn, addr_stub);
 
 using linalg_vector_norm_fn = void(*)(TensorIterator &, Scalar);
 DECLARE_DISPATCH(linalg_vector_norm_fn, linalg_vector_norm_stub);
-
-using unpack_pivots_fn = void(*)(
```

Comment:
The name of the file is unfortunate but BatchLinearAlgebra files, where this function declaration was moved, are mostly for LAPACK&BLAS wrappers. I'd keep `unpack_pivots_stub` in this file and have the git diff shorter.

I think it's also fine to leave it as it is and move the declarations if that's preferred.

Reply:
I might be wrong here, but imo these two files are just a legacy split which should not exist anymore. Both of them have LAPACK/BLAS wrappers. I just moved this function to be close to the other LU functions within `BatchedLinearAlgebra` as they are logically related.